akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -12,13 +12,15 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
 
   private lazy val serialization = SerializationExtension(system)
 
-  private val NoneSerialized = Array.empty[Byte]
+  private val ParameterlessSerialized = Array.empty[Byte]
```

Comment:
There actually is a `Array.emptyByteArray` in the standard library that we could reuse everywhere instead of keeping separate empty array instances.